src/entrypoints/background.ts

```
@@ -1,14 +1,14 @@
 import { defineBackground } from '#imports';
 import { onMessage } from '@/messaging';
-import { handleTranslate } from '@/messaging/handlers';
+import { handleAgent } from '@/messaging/handlers';
 import { createLogger } from '@/utils/logger';
 
 export default defineBackground(() => {
   const logger = createLogger('background');
-  logger.debug('Hello background!');
+  logger.debug`Hello background!`;
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)